kratos/geometries/point.h

```diff
@@ -150,13 +146,23 @@ class Point : public array_1d<double, 3>
     ///@name Operations
     ///@{
 
+    /**
+     * @brief This method computes the distance betwee this point and another one
+     * @param rOtherPoint The other point to compute the distance
+     * @return The distance between this and another point     
+     */
+    double Distance(const Point& rOtherPoint)
```

Comment:
const!

Reply:
I suppose this could be similarly using the CalculateDistance function.

Not sure about the efficiency, though.

Reply:
That function is in the geometry. A point is not a geometry (Point2D and Point3D)